### ISSUE TITLE
Streak count broken again by GH?

### ISSUE THREAD

**Issue by eddelbuettel** [Naramsim/GithubOriginalStreak#89]
Thanks again for fixing #86 last month. It looks like they are at it again as I back to 366 (== one year, rolling) rather that the high two-thousands it should show.

Can you work your magic again and catch up with whatever they changed?

**Comment by Naramsim** [comment]
Yes, we have another issue for that. I'll try

**Comment by eddelbuettel** [comment]
Should I just purge and re-install?  Did it "merely" forget to start date (years ago in my case) as I am now at 368 days :-/

**Comment by Naramsim** [comment]
When I will release the fix I will make sure to update your starting date

**Comment by eddelbuettel** [comment]
Thanks for fixing it again!

**Comment by Naramsim** [comment]
Hi @eddelbuettel, I hope this fix will continue to work.

**Comment by LeonTG** [comment]
@Naramsim Idk if this is the right place for this, but is it possible to fix my date as well?

Mine got reset back to a year too with this bug when it should be over 5 and a half years long

**Comment by Naramsim** [comment]
This is the right place!

It was 2015-09-12, am I right?

**Comment by LeonTG** [comment]
@Naramsim Yeah that's correct

**Comment by mojavelinux** [comment]
@Naramsim Mine was 2015-11-30. Could you update my start date as well? (Or is this something I should be able to set in the extension itself?)

**Comment by Naramsim** [comment]
Once there was a way, a popup would even show, but now that functionality doesn't work anymore. So I'll do it manually later

**Comment by mojavelinux** [comment]
Thanks for the info and for your time to make that update. I appreciate it!

**Comment by LeonTG** [comment]
> This is the right place!
> 
> It was 2015-09-12, am I right?

Hey I know this is an old issue, I just wanted to say that my streak on this has been reset again as of like 2 days ago, if it could be fixed again! Thanks in advance

**Comment by Naramsim** [comment]
Yes, we are aware, we lost the logic in a recent PR that was fixing other things. I don't have time to work on this extension anymore so I'm very limited.
